core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -1890,6 +1890,26 @@ export class LoggedOutUser extends BaseUser {
     );
   }
 
+  /**
+   * This function verifies that the classroom cards in classrooms page.
```

Comment:
Is something missing herein? Sounds incomplete.

Reply:
Oh yes, fixed it.